In [1]:
from scipy.io import netcdf
import calculations as calc

In [2]:
nc = netcdf.netcdf_file("data/era-int_pl_1979-2019-mm-l70-u.nc")

The coordinates of Singapur are 1N, 104E

In [3]:
longitudes = nc.variables["longitude"][:]
calc.convert_coordinates_to_grid(longitudes, 104)

148

In [4]:
latitudes = nc.variables["latitude"][:]
calc.convert_coordinates_to_grid(latitudes, 1)

126

So the values for Singapur are at the indices longitude = 148 and latitude = 126